benchmark/syclblas/CMakeLists.txt

```diff
@@ -45,7 +47,7 @@ endforeach()
 
 # Expression benchmarks: use source, not Library
 if(BUILD_EXPRESSION_BENCHMARKS)
-  set(extensions expression/reduction_rows.cpp)
+  set(extensions "")
```

Comment:
`BUILD_EXPRESSION_BENCHMARKS` doesn't build anything anymore - I'm still not sure of its exact purpose, @mehdi-goli might know more?

Reply:
The `BUILD_EXPRESSION_BENCHMARKS` has been added when the reduction was not part of the library build and it could only be used via header-only expression. I think This flag should be removed now that the reduction is officially part of the library extension.

Reply:
Thanks @ProGTX  for noticing it

Reply:
Good point, removed the flag from cmake files and `conanfile.py`.